protocol/stream/stream.go

```diff
@@ -25,12 +25,15 @@ package stream
 import (
 	"io"
 
+	internal "go.uber.org/thriftrw/internal/iface"
```

Comment:
We should avoid named import/aliasing if there isn't a conflict.

See also https://github.com/uber-go/guide/blob/master/style.md#import-aliasing